tests/integration/simple_integration_test.go

```diff
@@ -0,0 +1,164 @@
+		k8s.CreateNamespace(GinkgoT(), kubectlOptions, "k8ssandra")
+	})
+
+	AfterEach(func() {
+		kubectlOptions := k8s.NewKubectlOptions("", "", namespace)
+		options := &helm.Options{KubectlOptions: kubectlOptions}
+
+		// Verify everything was actually deleted.. I don't think Service is
+		helm.Delete(GinkgoT(), options, releaseNameCluster, true)
+		helm.Delete(GinkgoT(), options, releaseName, true)
+
+		// Delete all the resources from namespace
+		kubectlOptions = k8s.NewKubectlOptions("", "", "default")
+		k8s.DeleteNamespace(GinkgoT(), kubectlOptions, namespace)
+	})
+
+	Context("deploying", func() {
+		It("with default options", func() {
+			crdChartPath, err := filepath.Abs("../../charts/k8ssandra")
+			Expect(err).To(BeNil())
+
+			//kubectlOptions := k8s.NewKubectlOptions("", "", "default")
+			//k8s.CreateNamespace(GinkgoT(), kubectlOptions, namespace)
+			kubectlOptions := k8s.NewKubectlOptions("", "", namespace)
+			options := &helm.Options{KubectlOptions: kubectlOptions}
+
+			releaseName = fmt.Sprintf(
+				"demo-%s", strings.ToLower(random.UniqueId()))
+			helm.Install(GinkgoT(), options, crdChartPath, releaseName)
+
+			// k8s module has no select by label
+			clientset, err := k8s.GetKubernetesClientFromOptionsE(GinkgoT(), kubectlOptions)
+			pods, _ := clientset.CoreV1().Pods(namespace).List(context.Background(), metav1.ListOptions{LabelSelector: "name=cass-operator"})
+
+			Expect(len(pods.Items)).To(Equal(1))
+			k8s.WaitUntilPodAvailable(GinkgoT(), kubectlOptions, pods.Items[0].Name, 50, 500 * time.Millisecond)
+
+			// We could use also kubectl wait --for=condition=Ready pod -l name=cass-operator
+
+			// Deploy traefik
+			// helm install traefik traefik/traefik -n traefik --create-namespace -f docs/content/en/docs/topics/ingress/traefik/kind-deployment/traefik.values.yaml
+			//valuesPath, _ := filepath.Abs("../../docs/content/en/docs/topics/ingress/traefik/kind-deployment/traefik.values.yaml")
+			//helm.RunHelmCommandAndGetOutputE(GinkgoT(), options, "install", "traefik", "traefik/traefik", "-n", "traefik", "--create-namespace", "-f", valuesPath)
+
+			// Deploy the cluster
+			clusterChartPath, err := filepath.Abs("../../charts/k8ssandra-cluster")
+			Expect(err).To(BeNil())
+
+			releaseNameCluster = releaseName + "-cluster"
+			options = &helm.Options{
+				// Enable traefik to allow redirections for testing
+				SetValues: map[string]string{
+					"ingress.traefik.enabled": "true",
+					"ingress.traefik.monitoring.grafana.host": "grafana.localhost",
+					"ingress.traefik.monitoring.prometheus.host": "prometheus.localhost",
+					"ingress.traefik.repair.host": "repair.localhost",
+				},
+				KubectlOptions: k8s.NewKubectlOptions("", "", namespace),
+			}
+			helm.Install(GinkgoT(), options, clusterChartPath, releaseNameCluster)
+
+			// Wait for CassandraDatacenter to be ready..
+			k8s.RunKubectl(GinkgoT(), kubectlOptions, "wait", "--for=condition=Ready", "cassandradatacenter/dc1", "--timeout=300s")
```

Comment:
I wasn't really familiar with `kubectl wait` before I saw this. That is really nice :)

Reply:
>At this point I don't have a strong opinion on the method we choose.

Since we already invested in traefik for use with kind and since you already have it working in the test, I changed my mind and now have a stronger opinion. Traefik works for me 👍